server/src/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4468,6 +4472,16 @@ private boolean checkIfHostIsDedicated(HostVO host) {
         }
     }
 
+    private void checkIfHostOfVMIsInPrepareForMaintenanceState(Long hostId, Long vmId, String operation) {
```

Comment:
Hello @sureshanaparti 

I think this method can use a Javadoc. Since it's supposed to check something and doesn't return nothing, it might make someone have to open the code to check what does it do.